multi-vterm.el

```diff
@@ -83,7 +86,9 @@ If nil, this defaults to the SHELL environment variable."
 (defun multi-vterm-project ()
   "Create new vterm buffer."
   (interactive)
-  (if (project-current)
+  (if (if multi-vterm-projectile-installed-p
+          (projectile-project-p)
+        (project-current))
```

Comment:
I suppose that it still works when someone does not use `projectile`. Am I correct?

Reply:
It worked in my own testing!